lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -444,6 +444,41 @@ class BuildOpGroups : public calyx::FuncOpPartialLoweringPattern {
                         scf::ExecuteRegionOp executeRegionOp) const;
   LogicalResult buildOp(PatternRewriter &rewriter, CallOp callOp) const;
 
+  void handleCmpIOp(PatternRewriter &rewriter, CmpIOp cmpIOp, Operation *group,
+                    calyx::RegisterOp &condReg, calyx::RegisterOp &resReg,
+                    bool &isSequential) const {
+    isSequential = calyx::isPipeLibOpRes(cmpIOp.getLhs()) ||
+                   calyx::isPipeLibOpRes(cmpIOp.getRhs());
+    if (isSequential) {
```

Comment:
Nit: can we invert this to reduce indentation?

if (!isSequential) { return; }

Reply:
sounds good, we should!